src/components/AuthSettings.tsx

```diff
@@ -23,6 +23,13 @@ export default function AuthSettings(): ReactElement {
     }
   }, [loading, settings, user]);
 
+  const createFolder = async () => {
+    await firestoreCollection
+      .doc(user?.uid)
+      .set({ auth: { accessKeyId: "", secretAccessKey: "" } }, { merge: true });
+    location.reload();
```

Comment:
Name the function appropriately, so we know what it is. Also, we shouldn't do a hard refresh, since this is a single page application, so we should just refresh whatever data we need to.